static/js/reminder.js

```diff
@@ -80,7 +80,7 @@ export function schedule_message(request = compose.create_message_object()) {
             );
         } else {
             compose_error.show(
-                $t_html({defaultMessage: "Your reminder note is empty!"}),
+                $t_html({defaultMessage: "Your message is empty!"}),
                 $("#compose-textarea"),
             );
         }
```

Comment:
I think we probably want the same error message we have for a normal empty compose box?  

Also, this if/else block needs to be reworked; I think the outer if statement is useless and makes the code a lot harder to read and forces us to repeat all these conditionals, and should just be removed.

Reply:
Done.